Compatibility/Ludeon.RimWorld.Royalty/Languages/English/DefInjected/ThingDef/Plants_Wild.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<LanguageData>
+  <Plant_TreeAnima.label>bananima tree</Plant_TreeAnima.label>
+  <Plant_TreeAnima.description>A rare tree with warm, skin-smooth bark and long iridescent leaves which remind anyone lucky enough to see one of the sacred banana trees. Infused with bioluminescent micro-organisms, bananima trees develop a unique form of psychic symbiosis with surrounding lifeforms, allowing them to grow in a wide variety of biomes.\n\nIf a person (psycaster or not) meditates near an bananima tree, it will grow bananima grass around its base. Once enough grass is grown, it becomes possible to carry out a psychic linking ritual with the tree and upgrade a person's psychic powers. Only peoples intimiately connected to nature know the secret of this ritual. Tribal psycasters are also able to draw psyfocus from bananima trees while meditating to them.\n\nBananima trees' psychic properties are weakened if artificial structures are placed nearby. They refuse to be caged or studied, and must remain part of nature.\n\nMost tribes believe that bananima trees are not simply trees, but are rather the physical extremities of a single world spirit.</Plant_TreeAnima.description>
```

Comment:
```suggestion
  <Plant_TreeAnima.description>A rare tree with warm, skin-smooth bark and long iridescent leaves which remind anyone lucky enough to see one of the sacred banana trees. Infused with bioluminescent micro-organisms, bananima trees develop a unique form of psychic symbiosis with surrounding lifeforms, allowing them to grow in a wide variety of biomes.\n\nIf a person (psycaster or not) meditates near a bananima tree, it will grow bananima grass around its base. Once enough grass is grown, it becomes possible to carry out a psychic linking ritual with the tree and upgrade a person's psychic powers. Only peoples intimiately connected to nature know the secret of this ritual. Tribal psycasters are also able to draw psyfocus from bananima trees while meditating to them.\n\nBananima trees' psychic properties are weakened if artificial structures are placed nearby. They refuse to be caged or studied and must remain part of nature.\n\nMost tribes believe that bananima trees are not simply trees but are rather the physical extremities of a single world spirit.</Plant_TreeAnima.description>
```